### PR TITLE
Update wlcg_converter.py for gstream TPC to CMS

### PR DESCRIPTION
**Pull request by biozit**

### CODE REVIEW

**Comment by biozit** [comment]
I keep two functions on  wlcg_converter.py to keep it easy to customize

File: Collectors/DetailedCollector.py [hunk 1]
```diff
@@ -291,7 +291,7 @@ def process_gstream_tpc(self, gstream, addr, sid):
              event["size"] = event.pop("Size")
 
              if event["source"].startswith('/store') or event["source"].startswith('/user/dteam'):
-                  wlcg_packet = wlcg_converter.ConvertGstream(event)
+                  wlcg_packet = wlcg_converter.ConvertGstreamTPC(event)
```
Comment (djw8605) [inline Collectors/DetailedCollector.py:294]: Where else is the `ConvertGstream` function used?

File: Collectors/wlcg_converter.py [hunk 2]
```diff
@@ -199,6 +199,20 @@ def ConvertGstream(source_record):
 
     return source_record
     
+def ConvertGstreamTPC(source_record):
```
Comment (djw8605) [inline Collectors/wlcg_converter.py:202]: This function looks almost the exact same as the function above.  Would it be easier to have a single function, but pass in the "producer" as an argument to the function?

**Review by djw8605 — CHANGES_REQUESTED** [review]
A few comments.  I think you should consolidate the ConvertGstreamTPC and ConvertGstream functions into 1.

File: Collectors/wlcg_converter.py (hunk 2 above)
Comment (biozit) [reply Collectors/wlcg_converter.py]: sure

File: Collectors/DetailedCollector.py (hunk 1 above)
Comment (biozit) [reply Collectors/DetailedCollector.py]: sure

**Comment by biozit** [comment]
Do you have an "easy" way to test it?

**Review by biozit — COMMENTED** [review]
added the paramenter

**Review by biozit — COMMENTED** [review]
added the parameter

**Comment by biozit** [comment]
CMS montIT admins said is better to keep the producer and change the type.

**Review by biozit — COMMENTED** [review]
Done, keeping the producer and changing the type.

File: Collectors/wlcg_converter.py [hunk 3]
```diff
@@ -184,21 +184,25 @@ def Convert(source_record):
     return to_return
 
 
-def ConvertGstream(source_record):
+def ConvertGstream(source_record,tpc=False):
 
+    if tpc is True:
```
Comment (djw8605) [inline Collectors/wlcg_converter.py:189]:
```suggestion
    if tpc:
```

Comment (djw8605) [reply Collectors/wlcg_converter.py:189]: you could condense it to an Ternary:
```python
typeT = "tcp" if tcp else "metric"
```

But I don't love ternary.

**Review by djw8605 — COMMENTED** [review]
Just a quick comment.

File: Collectors/wlcg_converter.py (hunk 3 above)
Comment (biozit) [reply Collectors/wlcg_converter.py]: I like the frist one.

**Review by biozit — COMMENTED** [review]
done

**Comment by biozit** [comment]
The POD is ready to send the data

**Pull request merged**